automatic/protoc/README.md

```diff
@@ -0,0 +1,13 @@
+# chocolatey-protoc
+Chocolatey package for the prebuilt 
+[Protocol Buffers Compiler](https://developers.google.com/protocol-buffers/docs/downloads).
+
+## Overview
```

Comment:
Readme.md has standard outlook.
Remove `## Overview` and move that section to top.

Reply:
I was trying to resolve this when I realized that you have already done that. Thanks!